website/static/js/pages/share-embed-page.js

```diff
@@ -4,8 +4,14 @@ var pym = require('pym.js');
 
 var register = window.contextVars.share.urls.register;
 
-new pym.Parent(
+var pymParent = new pym.Parent(
     'share_registration_iframe',
     register,
     {}
 );
+
+function scrollToTop(data) {
+    window.scrollTo(0, 0);
+}
+
+pymParent.onMessage('scroll_top_now', scrollToTop)
```

Comment:
Yep! That's the message that's received from the Child page - which is in the SHARE Registration app here - https://github.com/erinspace/shareregistration/blob/master/provider_registration/templates/provider_registration/base.html#L28